authentication/src/components/Register.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import axios from 'axios';
+
+const URL = 'http://localhost:8000/api'
```

Comment:
Might want to consider looking into using a .env file to store environment variables.